app/src/display/widgets/Kconfig

```diff
@@ -25,6 +25,7 @@ config ZMK_WIDGET_OUTPUT_STATUS
     
 config ZMK_WIDGET_WPM_STATUS
     bool "Widget for displaying typed words per minute"
+    depends on ZMK_WPM
```

Comment:
I was actually suggesting if `CONFIG_ZMK_WIDGET_WPM_STATUS` it set it should auto-select `CONFIG_ZMK_WPM` as well instead of depending on `CONFIG_ZMK_WPM`

Reply:
Right, so instead of `depends on` you would use `select` here.

Reply:
Thanks folks - everyday's a learning day!

I think I've solved the logging warnings and changed it to select.

Reply:
Log warnings and select changes are good to go